scala/build/Dockerfile.mantic

```
@@ -26,13 +26,11 @@ ENV PATH="/venv/bin:$PATH"
 # Upgrade pip within the virtual environment
 RUN pip install --upgrade pip
 
-# Copy the requirements file into the container
-COPY scala/requirements.txt .
-
+# Python global dependencies requirements.txt file into the container
 # Install dependencies from the requirements file
+COPY scala/build/requirements.txt .
```

Hadolint

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)